Fixing Index and QVM files (#614)
rigetticomputing/pyquil: docs/source/index.rst

  with near-term quantum computers, see `Quantum Computing in the NISQ era and beyond <https://arxiv.org/abs/1801.00862>`_.

Our flagship product `Quantum Cloud Services <http://rigetti.com/qcs>`_ offers users an on-premise, dedicated access
point to our quantum computers. This access point is a fully-configured VM, which we call a Quantum Machine Image. A QMI

> is bundled with the same downloadable SDK mentioned above, and an admin command line interface (CLI), which is used for

Reviewer: ```suggestion
is bundled with the same downloadable SDK mentioned above, and a command line interface (CLI), which is used for
```
